src/functionalTest/resources/features/F-001 - Create IA Staff Role Assignments/F-001.feature

```diff
@@ -89,6 +173,34 @@ Feature: F-001 : Create Role Assignments for Caseworker Users
     And the response has all other details as expected
     And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments].
 
+  @S-001.03a
+  @FeatureToggle(DB:iac_wa_1_2=on) @FeatureToggle(EV:CASEWORKER_FTA_ENABLED=on)
```

Comment:
both `@S-001.03a `+ `@S-001.03b` should use flag: **iac_wa_1_3** as per `@S-001.03`